src/main/java/io/mosip/mimoto/constant/SwaggerLiteralConstants.java

```diff
@@ -0,0 +1,77 @@
+
+    /*  Credentials Controller */
+    public static final String CREDENTIALS_NAME = "Credentials Download using OpenId4VCI" ;
+    public static final String CREDENTIALS_DESCRIPTION = "All the Credentials Related Endpoints" ;
+    public static final String CREDENTIALS_DOWNLOAD_VC_SUMMARY = "Download Credentials as PDF" ;
+    public static final String CREDENTIALS_DOWNLOAD_VC_DESCRIPTION = "This endpoint allow you to retrieve all the Inji Wallet Properties" ;
+
+    /*  Credentials Share Controller */
+    public static final String CREDENTIALS_SHARE_NAME = "Credential Share" ;
+    public static final String CREDENTIALS_SHARE_DESCRIPTION = "All the Credential Download Endpoints" ;
+    public static final String CREDENTIALS_SHARE_HANDLE_SUBSCRIBED_EVENT_SUMMARY = "Notify through Web Sub Once Credential is downloaded" ;
+    public static final String CREDENTIALS_SHARE_HANDLE_SUBSCRIBED_EVENT_DESCRIPTION = "This endpoint allow Web Sub to callback once the credential is issued" ;
+    public static final String CREDENTIALS_SHARE_REQUEST_VC_SUMMARY = "request for credential issue" ;
+    public static final String CREDENTIALS_SHARE_REQUEST_VC_DESCRIPTION = "This endpoint allow you to request for credential issue" ;
+    public static final String CREDENTIALS_SHARE_REQUEST_VC_STATUS_SUMMARY = "polling for credential issue status" ;
+    public static final String CREDENTIALS_SHARE_REQUEST_VC_STATUS_DESCRIPTION = "This endpoint allow you to poll for credential issue status" ;
+    public static final String CREDENTIALS_SHARE_DOWNLOAD_VC_SUMMARY = "Download the credential using OTP Flow" ;
+    public static final String CREDENTIALS_SHARE_DOWNLOAD_VC_DESCRIPTION = "This endpoint allow you to download credential issued" ;
+
+    /*  IDP Controller */
+    public static final String IDP_NAME = "Wallet Binding" ;
+    public static final String IDP_DESCRIPTION = "All the Authorization Related Endpoints" ;
+    public static final String IDP_BINDING_OTP_SUMMARY = "Invoke OTP Request for Wallet Binding" ;
+    public static final String IDP_BINDING_OTP_DESCRIPTION = "This endpoint allow you to invoke OTP for wallet binding" ;
+    public static final String IDP_WALLET_BINDING_SUMMARY = "Wallet Binding" ;
+    public static final String IDP_WALLET_BINDING_DESCRIPTION = "This endpoint allow you to perform the wallet binding" ;
+    public static final String IDP_GET_TOKEN_SUMMARY = "Retrieve AccessToken for OIDC Flow" ;
+    public static final String IDP_GET_TOKEN_DESCRIPTION = "This endpoint allow you to retrieve the access token in exchange for authorization Code" ;
+
+    /*  Issuers Controller */
+    public static final String ISSUERS_NAME = "Issuers" ;
+    public static final String ISSUERS_DESCRIPTION = "All the Issuers Related Endpoints" ;
+    public static final String ISSUERS_GET_ISSUERS_SUMMARY = "Retrieve All Onboarded Issuers" ;
```

Comment:
should we mention about all enabled issuers?

Reply:
i dont think so, since its all internal to us, no need to explain it to api